### PR TITLE
SDIT-1244 Add tests and convert test file to kotlin

### PR DESCRIPTION
**Pull request by karenmillermoj**

### CODE REVIEW

File: src/main/java/uk/gov/justice/hmpps/prison/service/BookingService.java [hunk 1]
```diff
@@ -534,7 +534,6 @@ public OffenderBookingIdSeq getOffenderIdentifiers(final String offenderNo, fina
         return offenderIdentifier;
     }
 
-    @VerifyBookingAccess(overrideRoles = {"SYSTEM_USER"})
```
Comment (karenmillermoj) [inline src/main/java/uk/gov/justice/hmpps/prison/service/BookingService.java:537]: Not required as this is now only called from 2 places which both have the VerifyBookingAccess annotation

**Review by andymarke — APPROVED** [review]
lgtm

File: src/main/java/uk/gov/justice/hmpps/prison/repository/BookingRepository.java [hunk 2]
```diff
@@ -515,25 +515,6 @@ private static class OffenderBookingSeq {
         private Integer bookingSeq;
     }
 
-    public Page<ScheduledEvent> getBookingAppointments(final Long bookingId, final LocalDate fromDate, final LocalDate toDate, final long offset, final long limit, final String orderByFields, final Order order) {
```
Comment (andymarke) [inline src/main/java/uk/gov/justice/hmpps/prison/repository/BookingRepository.java:518]: so where has this code gone?

Comment (karenmillermoj) [reply src/main/java/uk/gov/justice/hmpps/prison/repository/BookingRepository.java:518]: It's was only called from endpoints that are not used.  It was called from my last PR (no longer used) but I forgot to remove it

**Pull request merged**